### PR TITLE
Update popular species button behaviour

### PR DESCRIPTION
**Pull request by msmanoj**
## Type

- [ ] Bug fix
- [ ] New feature
- [x] Improvement to existing feature

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-790

## Description
Clicking on a popular species button for a committed species will open the species homepage instead of removing the species.

## Deployment URL
http://favourite-species.review.ensembl.org

## Views affected
Species selector -> Popular species

### Other effects
- [ ] I have checked any requirements for Google Analytics
- [x] I have created any required tests
- [ ] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

### CODE REVIEW

File: src/ensembl/src/shared/helpers/urlHelper.ts [hunk 1]
```diff
@@ -31,6 +31,16 @@ type EntityViewerUrlParams = {
   view?: string | null;
 };
 
+type SpeciesHomepage = {
+  genomeId: string;
+};
+
+export const speciesHomepage = (params: SpeciesHomepage) => {
+  const speciesHomepageRootPath = '/species';
```
Comment (imransl) [inline src/ensembl/src/shared/helpers/urlHelper.ts:39]: When looking at these root paths being inside a function as a local variable, I wonder if it is better if we need to put them at a higher level in the same file.

File: src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx [hunk 2]
```diff
@@ -18,34 +18,32 @@ import React from 'react';
 import { connect } from 'react-redux';
 import classNames from 'classnames';
 import find from 'lodash/find';
+import { push } from 'connected-react-router';
 
+import Tooltip from 'src/shared/components/tooltip/Tooltip';
+import InlineSVG from 'src/shared/components/inline-svg/InlineSvg';
```
Comment (azangru) [inline src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx:24]: Why did you move these two imports? We usually try to group imports by their type; and now these two, which are components, are sitting together with helpres and hooks.

**Comment by azangru** [comment]
We have a bit of a naming confusion. I know that Andrea likes to talk about "species home page". In our code, it is currently referred to simply as "species page". Is there any reason to add the word "home" to it? Are there other species pages we are expecting? If not, I would advise against using the phrase 'species home page' in our code.

@ens-st3 might have an opinion.

File: src/ensembl/src/shared/helpers/urlHelper.ts (hunk 1 above)
Comment (azangru) [reply src/ensembl/src/shared/helpers/urlHelper.ts:39]: That would help if there were multiple functions interested in the same path segments, but there aren't.

File: src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx (hunk 2 above)
Comment (msmanoj) [reply src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx:24]: I've put it back. I'm still getting confused with the order. Can you please remind me the correct order once again?

**Comment by msmanoj** [comment]
> We have a bit of a naming confusion. I know that Andrea likes to talk about "species home page". In our code, it is currently referred to simply as "species page". Is there any reason to add the word "home" to it? Are there other species pages, as opposed to 'home', that we are expecting? If not, I would advise against using the phrase 'species home page' in our code.
> 
> @ens-st3 might have an opinion.

Fair enough. I have renamed it to `speciesPage` for now.

File: src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx (hunk 2 above)
Comment (azangru) [reply src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx]: I'm usually doing it like this:

1) libraries from node modules
2) non-components (redux actions or selectors, helpers, hooks), tend to group them by type
3) react components
4) typescript types (if not imported together with imports in one of the previous groups)
5) styles and, on occasion other static assets, such as images (not the svgs that we import as React components though; those go into the react components group).

What are your conventions?

Comment (msmanoj) [reply src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.tsx]: Thanks. I think I didn't separate non-component and react component and kept mixing them up. It's all clear now.

File: src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.test.tsx [hunk 3]
```diff
@@ -118,13 +123,19 @@ describe('<PopularSpeciesButton />', () => {
       );
     });
 
-    it('deletes committed species when clicked', () => {
+    it('opens species homepage when it is clicked', () => {
```
Comment (azangru) [inline src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.test.tsx:126]: homepage -> page?

Comment (msmanoj) [reply src/ensembl/src/content/app/species-selector/components/popular-species-button/PopularSpeciesButton.test.tsx]: That's done

**Review by azangru — APPROVED** [review]
